### PR TITLE
Login component

### PR DESCRIPTION
**Pull request by hshahwan**
## Checklist Criteria
- [ ] **Branch is up to date with master**
- [ ] **The project runs**
- - [x] You can download and install locally, and run the project locally
- - [x] All existing and new features run without bugs
- - [x] The build script runs without error
- - [ ] (If the project is already deployed to Heroku) the branch to be merged can deploy successfully to Heroku
- [x] **Code is consistent**
- - [x] Linter is used, no error messages
- - [x] Code is consistent with agreed team style
- - [x] Naming of files is consistent with project (e.g. all files named with camelCase)
- [x] **Tests are in place**
- - [x] Test file is in the same folder as the original file, with the extension .test.js
- - [x] Each new file written has a test file (where possible)
- [x] **Tests all pass**
- - [x] All tests should pass locally and on Travis
- - [ ] Edge cases have been considered in tests
- [ ] **Test coverage is maintained**
- - [ ] Project coverage stays > 95%
- - [ ] PR will not bring down coverage by more than 1%
- - [ ] Files and lines of code have been ignored if they cannot be tested
- - [ ] If you are stuck with writing tests, you have assigned @arrested-developer to the PR and added a comment to explain what help is needed
- [x] **No console logs**
- - [x] Console.log is only used for crucial system messages (e.g. which port server is on).
- - [x] Any console.logs used for debugging have been removed.
- - [x] Console.error is used in error handling.
- - [x] global.console.log and global.console.error are mocked in tests when needed to avoid littering the console

### CODE REVIEW

File: client/src/Components/Layouts/Login/Login.style.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  margin-left: 50px;
+  border: none;
+`
+
+export const HomeIcon = styled.img`
+  height: 30px;
+  width: 60px;
+  left: 0;
+`
+export const Input = styled.input`
```
Comment (arrested-developer) [inline client/src/Components/Layouts/Login/Login.style.js]: You can add attributes to your input like this:

```
export const Input = styled.input.attrs({
  type:"text",
})`
  border: none;
  height: 50px;
  border-radius: 10px;
  width: 240px;
  display: block;
  text-align: center;`
```

The attributes are listed in an object, and are written as you would usually write them in Javascript, as you probably know sometimes this is different from HTML

There is some documentation here:

https://www.styled-components.com/docs/basics#attaching-additional-props

Comment (arrested-developer) [reply client/src/Components/Layouts/Login/Login.style.js]: you can only set the attributes that you know are always the same, but you could create a sub-component from this and set attributes on it:

```
const TextInput = styled(Input).attrs({
  type="text",
})``
```

If it is an attribute that will always change, then there is no point setting it in the styled component, better to do this in the JSX

**Comment by hshahwan** [comment]
@thejoefriel @dupreesi  Check it again and tell me if there any problem :D
